packages/hooks/src/test/index.test.js

```diff
@@ -193,6 +193,13 @@ test( 'cannot add filters with invalid namespaces', () => {
 	);
 } );
 
+test( 'cannot add filters with namespace containing backslash', () => {
+	addFilter( 'hook_name', 'i\n\v\a\l\i\d\a\n\a\m\e', () => null );
```

Comment:
Fun fact is that `invalidn\amae` will pass. I have no idea how to fix that. We can open a follow-up if we are really concerned about it.